### DISCUSSION TITLE
how to start monailabel server on aws ec2 instance.

### DISCUSSION THREAD

**Discussion by anuragp2018** [Project-MONAI/MONAILabel#1673] [General]
please guide me how to run monailabel server on ec2 instance i already install monailabel on server but unable to start the server.
and also please guide me how can i access server remotely on another machine.

**Comment by diazandr3s** [comment]
Hi @anuragp2018,

Can you please provide more details? Is there a reason why you can't start the MONAI Label server? Is there an error you can share?
Once you start the server by following these instructions: https://github.com/Project-MONAI/MONAILabel?tab=readme-ov-file#step-5-start-monai-label-server-and-start-annotating
you should be able to connect to it using MONAI Label extension in Slicer by providing the IP address of the AWS instance.

Hope this helps,
